### DISCUSSION TITLE
Aktif Kullanıcılara İbax Destegi

### DISCUSSION THREAD

**Discussion by Majezikonline** [IBAX-io/go-ibax#464] [Ideas] [answered]
### Clear and concise description of the problem

Proje yeni ve forumdakilerin çoğu Türk. Herkes ibax toplamak için uğraşıyor. Fakat kime ne kadar çıkacağı belli değil. Forumda destek veren, öneri sunan, platforma tamamen yararlı yazıları olan kişilerin PLATFORM SAHİPLERİNCE ÖN PLANDA tutulması gerekir👍 

### Suggested solution

Her yararlı başlık ve içeriğe 10-100 ibax coin verilmesi

### Alternative

Forumda kullanıcların yararına yazılar yazan forum üreticilerine 10-100 ibax fazla gelirse; rütbe ve kademe şartı getirilmeli. 

### Validations

- [X] Read the [docs](https://docs.ibax.io).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.

**Comment by HalloKyle** [comment, marked as answer]
We have provided a complete set of GitHub activities and Ranking System, please pay attention to the rules.

**Reply by Majezikonline** [reply]
I'm not just talking about the time period until ibax support ends.  I'm talking about the times after the ibax is distributed. :)♥️
